app.go

```diff
@@ -681,7 +744,7 @@ func (app *App) executeInvokes() error {
 	for _, i := range app.invokes {
 		fn := i.Target
 		fname := fxreflect.FuncName(fn)
-		app.logger.Printf("INVOKE\t\t%s", fname)
+		fxlog.Info("invoke", fxlog.F("funcname", fname)).Write(app.log)
```

Comment:
we should try to go for consistent key naming patterns. if we're using "return value" (or the camelCase variant), then let's use "function" or "function name"  (or its camelCase variant). otherwise, we'd want to use retval which is less helpful.